packages/clay-shared/src/useFocusManagement.ts

```diff
@@ -232,6 +271,19 @@ export function useFocusManagement(scope: React.RefObject<null | HTMLElement>) {
 				// when have React Portal.
 				if (nextElement) {
 					setInteractionObserver(nextElement);
+				} else {
+					// When the next element is not in scope, we check for the next
+					// focusable element in the document that is not in the React tree.
+					// This is particularly used for Portals.
+					const docElements = collectDocumentFocusableElements();
```

Comment:
Perhaps we should make the same case for when it is backwards? use of shift + tab.